dev-guide/api-conventions.md

```diff
@@ -255,3 +280,18 @@ Following this convention has the disadvantage that it deviates from what users
 may be accustomed to from upstream APIs, but it has the advantage that it avoids
 ambiguity and the need for API consumers to resolve an API version and kind to
 the resource group and name that identify the resource.
+
+## FAQs
+
+### My proposed design looks like an existing API we have, why am I being told that it must be changed?
+
+There are a few reasons why an API reviewer might want you to change the proposed design of your API addition.
+
+When the API looks similar to an existing API, there are a couple of important things to bear in mind.
+
+Firstly, not all APIs in OpenShift have been through the API review process, therefore, especially early
+in the OpenShift 4 lifecycle, many APIs were shipped that were not compliant with the conventions.
+If you API looks like an existing API that is not compliant, it must be updated to be compliant.
```

Comment:
For me it's not quite clear what must be updated. Can this sentence be more specific?

Reply:
Updated, is it clearer with the new wording?